core/src/model/Entity.java

```diff
@@ -23,6 +23,15 @@ public void setCollisionResponse(CollisionFilter collisionResponse) {
         this.collisionResponse = collisionResponse;
     }
 
+    /**
+     * Abstract constructor
+     * @param x entity x coordinate position
+     * @param y entity y coordinate position
+     * @param height entity height
+     * @param width entity width
+     * @param damage entity damage
+     * @param world the world entity resides in
```

Comment:
It should probably be mentioned that world contains information about collision boxes.